src/components/RegionItem/RegionItem.style.tsx

```diff
@@ -51,9 +65,13 @@ export const SharedWrapperStyles = css`
   display: flex;
   align-items: center;
   border: 1px solid ${COLORS.LIGHTGRAY};
-  width: 330px;
-  padding: 1.25rem 1.75rem 1.25rem 1.25rem;
+  max-width: 296px;
+  padding: 1rem 1.25rem 1rem 1.25rem;
   border-radius: 4px;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    width: 296px;
```

Comment:
nit: might be good to make this a constant with above

Reply:
good idea, thanks